### ISSUE TITLE
Plot methods in snek5000/output/print_stdout.py

### ISSUE THREAD

**Issue by paugier** [snek5000/snek5000#81]
Nek5000 logs are quite complicated (and for me mysterious). I get something like:

```
Starting time loop ...

     DT/DTCFL/DTFS/DTINIT   0.100E-01   0.100E-01   0.000E+00   0.100E-01
Step      1, t= 1.0000000E-02, DT= 1.0000000E-02, C=  0.000 0.0000E+00 0.0000E+00
             Solving for Hmholtz scalars
  Temperature/Passive scalar solution
 New CG1-tolerance (RINIT*epsm) =    1.1153045821067478E-014   0.0000000000000000     
          1  Hmholtz TEMP       1   1.1153E-01   7.3922E-05   1.0000E-07   1.0000E+02   F
          1  Hmholtz TEMP       2   1.7017E-03   7.3922E-05   1.0000E-07   1.0000E+02   F
          1  Hmholtz TEMP       3   3.7470E-05   7.3922E-05   1.0000E-07   1.0000E+02   F
          1  Hmholtz TEMP       4   1.1114E-06   7.3922E-05   1.0000E-07   1.0000E+02   F
          1  Hmholtz TEMP       5   1.9912E-08   7.3922E-05   1.0000E-07   1.0000E+02   F
          1  Hmholtz TEMP       4   1.9912E-08   1.1153E-01   1.0000E-07
          1  Scalars done  1.0000E-02  1.0160E-03
             Solving for fluid
          1  PRES gmres         0   0.0000E+00   0.0000E+00   1.0000E-05   0.0000E+00   8.1685E-05    F
          1  Helmh3 fluid       0   0.0000E+00   0.0000E+00   1.0000E-07
             L1/L2 DIV(V)           0.0000E+00   0.0000E+00
             L1/L2 QTL              0.0000E+00   0.0000E+00
             L1/L2 DIV(V)-QTL       0.0000E+00   0.0000E+00
          1  Fluid done  1.0000E-02  7.0113E-04
Step      2, t= 2.0000000E-02, DT= 1.0000000E-02, C=  0.000 3.4612E-03 3.4612E-03
             Solving for Hmholtz scalars
          2  Hmholtz TEMP       4   6.1961E-09   1.6318E-01   1.0000E-07
          2  Scalars done  2.0000E-02  3.2448E-04
             Solving for fluid
          2  PRES gmres         6   7.4452E-06   2.4753E-03   1.0000E-05   6.4027E-04   9.9262E-04    F
          2  Helmh3 fluid       3   3.3066E-08   4.4121E-05   1.0000E-07
             L1/L2 DIV(V)          -6.4656E-20   1.5455E-03
             L1/L2 QTL              0.0000E+00   0.0000E+00
             L1/L2 DIV(V)-QTL      -6.4656E-20   1.5455E-03
          2  Fluid done  2.0000E-02  1.9669E-03
Step      3, t= 3.0000000E-02, DT= 1.0000000E-02, C=  0.001 6.6836E-03 3.2224E-03
             Solving for Hmholtz scalars
          3  Hmholtz TEMP       4   3.4024E-09   1.9467E-01   1.0000E-07
          3  Scalars done  3.0000E-02  3.1270E-04
             Solving for fluid
          3  PRES gmres         7   4.5838E-06   3.9343E-03   1.0000E-05   6.8044E-04   1.0463E-03    F
          3  Helmh3 fluid       3   1.9763E-08   4.4491E-05   1.0000E-07
             L1/L2 DIV(V)          -1.3031E-19   7.3944E-04
             L1/L2 QTL              0.0000E+00   0.0000E+00
             L1/L2 DIV(V)-QTL      -1.3031E-19   7.3944E-04
          3  Fluid done  3.0000E-02  1.8685E-03
[...]
```

It would be great to also print the clock time for each time step so that we could get figures as `plot_clock_times` in https://foss.heptapod.net/fluiddyn/fluidsim/-/blob/branch/default/fluidsim/base/output/print_stdout.py#L222. It is really very useful.

I guess that other plots representing the data printed by Nek5000 could also be useful.

**Comment by paugier** [comment]
Surprisingly, I can't find documentation on what is printed by Nek5000 :slightly_frowning_face:

**Comment by akhoubani** [comment]
We could look at https://github.com/Nek5000/Nek5000/blob/master/core/drive2.f

**Comment by ashwinvis** [comment]
Good find @akhoubani. To be more precise `TTIME_STP` seems to be what you want:

https://github.com/Nek5000/Nek5000/blob/71669e62586c7786a936a29306e4f37a0b9c1490/core/drive2.f#L127-L136

I think plotting number of pressure iterations (`iter`) and divergence (`divex`) is also a useful diagnostic - something's wrong if it goes up. There are different pressure solvers, but the default `gmres` should do for now:

https://github.com/Nek5000/Nek5000/blob/3e3eb9ac9b0099ddb1f385be04e3784cde4a58a8/core/gmres.f#L537-L540

**Comment by paugier** [comment]
Fixed by #121 !
